CHANGELOG.md

```diff
@@ -52,7 +60,7 @@
 ### Features
 
 1. [#387](https://github.com/influxdata/influxdb-client-js/pull/387): Add FluxScriptInvocationAPI.
-1. [#387](https://github.com/influxdata/influxdb-client-js/pull/387): Add invocableScript.js example.
+1. [#387](https://github.com/influxdata/influxdb-client-js/pull/387): Add invokableScript.js example.
```

Comment:
I would prefer to not alter the past, it was called invocable scripts at that moment
```suggestion
1. [#387](https://github.com/influxdata/influxdb-client-js/pull/387): Add invocableScript.js example.
```